sbin/newfs_msdos/newfs_msdos.8

```diff
@@ -159,6 +160,9 @@ Number of hidden sectors.
 Number of reserved sectors.
 .It Fl s Ar total
 File system size.
+.It Fl T Ar timestamp
+Specify a timestamp to be used for filesystem creation so that it can be consistent for reproducible builds.
```

Comment:
This should be wrapped at < 80 cols
There is a FreeBSD port `textproc/igor` (or pkg `igor`) which will check man pages and other docs for style issues.